Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.10.5-slim
+LABEL maintainer="davidkrivko@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+ENV API_Key e89d80d78e294b5e93f111857223012

```

Comment:
You should remove API_KEY from Dockerfile. It will pass with -e key

![image](https://user-images.githubusercontent.com/104218550/211519475-f745dd0a-25ad-4f05-a2dd-59d2917d9844.png)

Reply:
remove api key from ur dockerfile